examples/blocks/src/custom/index.html

```diff
@@ -77,14 +80,15 @@
                 const name = metadata.value;
                 const span = document.createElement("span")
                 const img = document.createElement("img");
-                img.onload = () => {
-                    img.onload = undefined;
-                    const data = getBase64Image(img);
-                    img.src = data;
-                }
+<!--                img.onload = () => {-->
```

Comment:
Looks like an HTML comment here and not JS?

Reply:
True, however in testing it was not a blocker, but just to be clear I have removed the whole commented out section.